extensions/lastpass/src/utils/cli.ts

```diff
@@ -0,0 +1,80 @@
+import { exec, ExecException } from "child_process";
+
+export type Account = {
+  name: string;
+  id: string;
+  username: string;
+  password: string;
+  url: string;
+  lastModified: Date;
+  lastTouch: Date;
+  fullname: string;
+  group: string;
+  note?: string;
+};
+
+const serializeFromJson = (jsonArray: string): Account[] => {
+  const array: any[] = JSON.parse(jsonArray);
+  const res = array.map(
+    (obj) =>
+      ({
+        ...obj,
+        lastModified: new Date(parseInt(obj.last_modified_gmt, 10) * 1000),
+        lastTouch: new Date(parseInt(obj.last_touch, 10) * 1000),
+      } as Account)
+  );
+  return res;
+};
+
+const execute = async (
+  command: string,
+  opts: { log: { stdout: boolean; stderr: boolean } } = { log: { stdout: false, stderr: false } }
+) => {
+  const wrappedCommand = `zsh -l -c '${command}'`;
```

Comment:
is it necessary to wrap the command? Why not invoke `lpass` directly?

Reply:
because by default it takes `/bin/sh`, which is not people usually use 
![CleanShot 2023-02-13 at 23 06 38@2x](https://user-images.githubusercontent.com/21224705/218574709-497a7272-b2c9-488b-a526-0b54c0af5d87.png)

Reply:
I've steal that from `displayplacer` extension

Reply:
got it. The problem is, I don't use zsh, but [fish](https://fishshell.com/). 😅 

Unfortunately I can't get it to work without modifying extension code. What _does_ work is either:

- changing `execute` s.t. it uses `fish -l -c <>` instead of `zsh -l -c <>`
- changing occurrences of `lpass` to `/opt/homebrew/bin/lpass`

<img width="1670" alt="image" src="https://user-images.githubusercontent.com/1155589/218694657-2f07245d-feda-4b57-abb1-c58ee55ce557.png">

In general though, I'd say it's probably better to add a `cliPath` preference (similar to [BitWarden](https://github.com/raycast/extensions/blob/main/extensions/bitwarden/package.json#L103-L110)), which could also remove the need for dropping into `zsh` (or whatever other shell). 😸

Reply:
i think using `zsh` in this case is easier from development point of view ( we don't need to consider lots of shells with their specifics ) and since its in macos by default ( so it should work for everyone )...

thanks for input, it helped me to moreless reproduce issue and find a good solution which should suite everyone 🌻